components/automate-ui/src/app/components/projects-dropdown/projects-dropdown.component.ts

```diff
@@ -26,15 +27,22 @@ export class ProjectsDropdownComponent implements OnInit {
   @Input() disabled = false;
 
   // Emits the array of projects to the parent component
-  @Output() onSelection = new EventEmitter<any>();
+  @Output() onSelection = new EventEmitter<Array<Project>>();
+
+  active = false;
+  showError = false;
```

Comment:
I do not think `showError` and its associated template code are necessary. (And I was not able to trigger it, anyway.) But the point of it, per the associated error message, is that it requires an item to be selected, and that is not true.

Reply:
this component is going to be used in the future for required situations, in which we will need the error.

Reply:
Makes sense. I withdraw my comment then. 😁